__tests__/accessiblity_audit.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-env jest */
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.testPort
+const { AxePuppeteer } = require('axe-puppeteer')
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+const thingsToExclude = [
+  // axe reports there is "no label associated with the text field", when there is one.
+  ['#app-site-search__input'],
+  // axe reports that the phase banner is not inside a landmark, which is intentional.
+  ['.app-phase-banner__wrapper']
```

Comment:
Just an observation: I wasn't able to reproduce this error locally when I removed `['.app-phase-banner__wrapper']` from `thingsToExclude` (unlike `['#app-site-search__input']` which threw up an error when removed from `thingsToExclude`). 

But I can see the error when running Axe in the browser.

Reply:
I'll investigate and do a follow up PR if it makes sense to thanks Hanna.